program-runtime/src/serialization.rs

```diff
@@ -764,14 +764,30 @@ mod tests {
                     transaction_context,
                     transaction_accounts
                 );
-                invoke_context
-                    .transaction_context
-                    .configure_next_instruction_for_tests(
-                        0,
-                        instruction_accounts,
-                        &instruction_data,
-                    )
-                    .unwrap();
+                if instruction_accounts.len() > MAX_ACCOUNTS_PER_INSTRUCTION {
+                    // Special case implementation of configure_next_instruction_for_tests()
+                    // which avoids the overflow when constructing the dedup_map
+                    // by simply not filling it.
+                    let dedup_map = vec![u8::MAX; u8::MAX as usize + 1];
```

Comment:
Can you use `MAX_ACCOUNTS_PER_TRANSACTION` here instead, to be more clear?

```suggestion
                    let dedup_map = vec![u8::MAX; MAX_ACCOUNTS_PER_TRANSACTION + 1];
```